docs/ops/sequence/AUGRUCell_dev.md

```diff
@@ -0,0 +1,144 @@
+# AUGRUCell  {#openvino_docs_ops_sequence_AUGRUCell_dev}
+
+**Versioned name**: *AUAUGRUCell-dev*
```

Comment:
what is `dev`? Will we have versions for internal operations etc.?
@jane-intel, please share your thoughts and idea to maintain internal operations

Reply:
I've added `dev` to somehow distinguish it from the regular versioning.

Reply:
I see. We seem to have no need in versioning like dev-2 or so. We will just update it under the same tag, right?

Reply:
This is how I understand this approach, we can modify this spec and devlop/update the operation as long we keep it out of the opset.